core/controller.go

```diff
@@ -268,6 +303,51 @@ func (c *Controller) StopStreamHandler(w http.ResponseWriter, r *http.Request, p
 	w.WriteHeader(http.StatusOK)
 }
 
+func (c *Controller) StartPreloadStream(Name string, Uri string) {
+	var dto StreamDTO
+
+	dto.URI = Uri
+	logrus.Debugf("%s is being initialized", dto.URI)
+
+	_, knownStream := c.index[dto.URI]
+	if knownStream {
+		return
+	}
+
+	stream, id := streamer.NewStream(
+		dto.URI,
+		c.spec.StoreDir,
+		c.spec.KeepFiles,
+		c.spec.Audio,
+		streamer.ProcessLoggingOpts{
+			Enabled:    c.spec.ProcessLogging.Enabled,
+			Compress:   c.spec.ProcessLogging.Compress,
+			Directory:  c.spec.ProcessLogging.Directory,
+			MaxAge:     c.spec.ProcessLogging.MaxAge,
+			MaxBackups: c.spec.ProcessLogging.MaxBackups,
+			MaxSize:    c.spec.ProcessLogging.MaxSize,
+		},
+		25*time.Second,
+	)
+
+	streamname := id
```

Comment:
Can we camelcase this please to `streamName`? Makes the code more consistent and easier to read

Reply:
changed